be/src/vec/functions/function_bitmap.cpp

```diff
@@ -1168,10 +1164,6 @@ class FunctionBitmapSubs : public IFunction {
 
         default_preprocess_parameter_columns(argument_columns, col_const, {1, 2}, block, arguments);
 
-        for (int i = 0; i < 3; i++) {
```

Comment:
same problem with `RangeImplUtil`, it seems you remove some update logic to `res_null_map`, but actually `res_null_map` is still being used in the code.